### PR TITLE
[MU4] Fixed code style

### PR DESCRIPTION
**Pull request by igorkorsukov**
Added framework and mu4 dirs to check code style.
And fixed code style

### CODE REVIEW

File: framework/ui/internal/uiengine.h [hunk 1]
```diff
@@ -38,11 +38,11 @@ class UiEngine : public QObject, public IUiEngine
 {
     Q_OBJECT
 
-    Q_PROPERTY(QmlTheme* theme READ theme NOTIFY themeChanged)
-    Q_PROPERTY(QmlToolTip* tooltip READ tooltip CONSTANT)
+    Q_PROPERTY(QmlTheme * theme READ theme NOTIFY themeChanged)
+    Q_PROPERTY(QmlToolTip * tooltip READ tooltip CONSTANT)
```
Comment (Jojo-Schmitz) [inline framework/ui/internal/uiengine.h:42]: adding a space here???

File: framework/ui/view/interactiveprovider.cpp [hunk 2]
```diff
@@ -87,7 +87,7 @@ void InteractiveProvider::fillData(QmlLaunchData* data, const UriQuery& q) const
     data->setValue("modal", params.value("modal", ""));
 }
 
-void InteractiveProvider::fillData(QObject *object, const UriQuery &q) const
+void InteractiveProvider::fillData(QObject* object, const UriQuery& q) const
```
Comment (Jojo-Schmitz) [inline framework/ui/view/interactiveprovider.cpp:90]: but(re)moving it here?

File: framework/ui/internal/uiengine.h (hunk 1 above)
Comment (igorkorsukov) [reply framework/ui/internal/uiengine.h:42]: Unfortunately, the tool does not recognize this expression :(

Comment (Jojo-Schmitz) [reply framework/ui/internal/uiengine.h:42]: Great :-(

**Pull request merged**